packages/editor/src/components/editor-canvas/index.js

```diff
@@ -299,6 +301,7 @@ function EditorCanvas( {
 			styles={ styles }
 			height="100%"
 			iframeProps={ {
+				className: classnames( { 'has-history': hasHistory } ),
```

Comment:
Shouldn't we apply the same classname in template mode?

Reply:
As mentioned above we will have a follow-up PR that will handle all of the template mode migration to this new approach.